corehq/pillows/mappings/domain_mapping.py

```diff
@@ -246,6 +246,7 @@
                 'sms_mobile_worker_registration_enabled': {'type': 'boolean'},
                 'snapshot_time': {'format': "yyyy-MM-dd||yyyy-MM-dd'T'HH:mm:ssZZ||yyyy-MM-dd'T'HH:mm:ss.SSSSSS||yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'||yyyy-MM-dd'T'HH:mm:ss'Z'||yyyy-MM-dd'T'HH:mm:ssZ||yyyy-MM-dd'T'HH:mm:ssZZ'Z'||yyyy-MM-dd'T'HH:mm:ss.SSSZZ||yyyy-MM-dd'T'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss.SSSSSS||mm/dd/yy' 'HH:mm:ss",
                                   'type': 'date'},
+                'snapshot_head': {'type': 'boolean'},
```

Comment:
You also need to update the index name at the top of the file to trigger a reindex.  Luckily the domains reindex is so quick I think we can do it with the normal deploy.